164859698 layout the boilerplate

willeswa opened the pull request:
### What does this PR do?

Set up the project boilerplate

### Description of Task to be completed?

[x] Use create-react-app to bootstrap the layout
[x] Install the Airbnb as the default linting tool

### How should this be manually tested?

Access the repository on GitHub from [here](https://github.com/deferral/ah-django-frontend.git)
Switch to the branch ch-setup-boilerplate-164859698
You should see a directory called app with all the setup files

### Any background context you want to provide?

The create-react-app has the essentials that sets up the basis of developing the frontend app

### What are the relevant pivotal tracker stories?

[#164859698](https://www.pivotaltracker.com/story/show/164859698)

philophilo commented on app/README.md:
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```
This readme should be in the root directory

philophilo commented on app/public/manifest.json:
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Change the names here

philophilo commented on app/README.md:
```diff
@@ -0,0 +1,68 @@
+
+### `npm run build` fails to minify
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
```
Get rid of the parts that are not important have the project running. Generic information regarding React is not important

willeswa committed:
164859698-chore(setup): layout the boilerplate

- bootstrap using the create-react-app
- install the airbnb linting

[Delivers #164859698]

wangonya commented on package.json:
```diff
@@ -0,0 +1,32 @@
+    "not op_mini all"
+  ],
+  "devDependencies": {
+    "eslint": "^5.3.0",
```
~My system is complaining that this version of eslint is a bit ancient. It requires `5.12.0`. Any reason you chose to go with this? Or was it the default?~

~Never mind. The problem was local.~

wangonya replied:
After fixing my local settings, I realized npm was still giving me warnings. We are having two different versions of `eslint` installed on the same project. I don't know if everything is fine on your side, but things are not working well on mine with both. I will remove the lower version.

![Screenshot 2019-04-16 at 17 19 27](https://user-images.githubusercontent.com/19375569/56217448-f0d88680-606b-11e9-90a9-0eea60dd5211.png)

philophilo commented on README.md:
```diff
@@ -1 +1,18 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
```
Use your own words to describe your project, how to run it... 
what do you run to start the project... Make it easy for a 5-year-old 😁
